src/traces/sankey/select.js

```diff
@@ -0,0 +1,36 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = function selectPoints(searchInfo, selectionTester) {
+    var cd = searchInfo.cd;
+    var selection = [];
+    var fullData = cd[0].trace;
+
+    var nodes = fullData._sankey.graph.nodes;
+
+    for(var i = 0; i < nodes.length; i++) {
+        var node = nodes[i];
+        if(node.partOfGroup) continue; // Those are invisible
+
+        // TODO: decide on selection criteria, using centroid for now
+        var pos = [(node.x0 + node.x1) / 2, (node.y0 + node.y1) / 2];
```

Comment:
Voting :+1: on centroid.

Reply:
Ok :+1:  I :hocho: the TODO in 20d8c92d8454cc804e71b84a90469df807d14898

Maybe we should add a visual cue to let the user know a given node is now part of a selection :thinking: This could also be added later on.